webmessaging/message-channels/close.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+async_test(t => {
+    const c = new MessageChannel();
+    c.port1.onmessage = t.unreached_func('Should not have delivered message');
+    c.port1.close();
+    c.port2.postMessage('TEST');
+    setTimeout(t.step_func_done(), 100);
```

Comment:
Does this timeout value rely on anything in the spec? (sorry if I missed that)
If not, is there any chance you can make it a constant at the top of the script, so it's easy to tweak either way?

Alternatively, is it possible to have a more robust testing scheme that avoids the timeout altogether? For example, you could create two MessageChannels, close the first one, send messages to both (in that order), and end the test when you get the message on the second channel.

I read https://html.spec.whatwg.org/multipage/comms.html#implementation-advice very briefly, but I think it entails that you'd get the messages in order, since they'd be queued on the same task source?